apps/patient/src/components/BrandedPharmacyCard.tsx

```diff
@@ -22,7 +26,8 @@ interface Props {
 const PHARMACY_BRANDING = {
   [process.env.REACT_APP_CAPSULE_PHARMACY_ID as string]: {
     logo: capsuleLogo,
-    description: 'FREE Delivery within 1-2 Days'
+    description: 'FREE Delivery within 1-2 Days',
+    description2: 'FREE Same Day Delivery'
```

Comment:
can these be more descriptive names? like `descriptionSameDay` and `descriptionNextDay`? it'll make reading the ternary below a lil easier

Reply:
yep!